docs/content/documentation/systems/Functions/framework/PiecewiseBilinear.md

```diff
@@ -1,8 +1,14 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # PiecewiseBilinear
+
 !description /Functions/PiecewiseBilinear
 
+## Description
+
+The `PiecewiseBilinear` function reads a csv file and interpolates values based on the data in the file.  The interpolation is based on x-y pairs.  If `axis` is given, time is used as the y index.  Either `xaxis` or `yaxis` or both may be given.  Time is used as the other index if one of them is not given.  If `radius` is given, `xaxis` and `yaxis` are used to orient a cylindrical coordinate system, and the x-y pair used in the query will be the radial coordinate and time.
```

Comment:
break this into multiple lines. I ask you to do this so in the future when it is edited the diff is easier to view when there are more lines.